### PR TITLE
Add labels to two buttons that were missing them

### PR DESCRIPTION
**Pull request by techknowlogick**
Thanks to devinprater on mastodon for raising the point about these two buttons missing labels.

### CODE REVIEW

File: templates/repo/home.tmpl [hunk 1]
```diff
@@ -124,7 +124,7 @@
 				{{if eq $n 0}}
 					<div class="ui action tiny input" id="clone-panel">
 						{{template "repo/clone_buttons" .}}
-						<button id="download-btn" class="ui basic jump dropdown icon button tooltip" data-content="{{.locale.Tr "repo.download_archive"}}" data-position="top right">
+						<button id="download-btn" class="ui basic jump dropdown icon button tooltip" data-content="{{.locale.Tr "repo.download_archive"}}" data-position="top right" aria-label="{{.locale.Tr "repo.download_archive"}}">
```
Comment (silverwind) [inline templates/repo/home.tmpl]: This button would change to a regular menu with https://github.com/go-gitea/gitea/pull/19999, so I'm not sure how much sense it makes to create this future merge conflict now :)

Comment (techknowlogick) [reply templates/repo/home.tmpl]: Fair enough, I'll remove it from this PR if it'll get overridden in the other one. I'll include it in the backport though

Comment (silverwind) [reply templates/repo/home.tmpl]: Yeah, keep it in 1.17. I expect that PR to land soon to 1.18.

Comment (techknowlogick) [reply templates/repo/home.tmpl]: k, updated :)

**Review by silverwind — APPROVED** [review]

**Review by noerw — APPROVED** [review]

**Comment by 6543** [comment]
:rocket:

**Comment by 6543** [comment]
-> #20524

**Pull request merged**